packages/frontend/src/components/message/MessageListAndComposer.tsx

```diff
@@ -98,6 +100,15 @@ export default function MessageListAndComposer({ accountId, chat }: Props) {
   const { openDialog, hasOpenDialogs } = useDialog()
   const { sendMessage } = useMessage()
 
+  // Create a store instance and share it between
+  // useMessageList and useDraft to avoid redundant data loading
```

Comment:
IDK if this comment is much useful. It's clear from the code that `messageListStore` is shared.